Winium/Winium.Mobile.Common/ConnectionInformation.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace Winium.StoreApps.Common
+﻿namespace Winium.Mobile.Common
 {
     public class ConnectionInformation
     {
         #region Public Properties
 
-        public const string FileName = ".Winium.StoreApps.ConnectionInformation.json";
+        public const string FileName = ".Winium.Mobile.ConnectionInformation.json";
```

Comment:
I wouldn't say that this is backwards compatibility issue. Public API wasn't changed. Do we support work for different driver/server versions?
Will have a look at this case later.